<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2" rowspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </item>
    <item row="3" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_4"/>
    </item>
    <item row="3" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_5"/>
    </item>
    <item row="4" column="3" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_6"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
